domains/hymoonz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "CNAME": "hymoonz.github.io"
```

Comment:
404 CNAME.

Reply:
done sir

Reply:
Redirect loop now.

Reply:
That's just the beginning to create my profile

Reply:
No sir

Reply:
I've changed it